Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -445,6 +445,40 @@ + (BOOL)date:(NSDate*)date isBetweenDate:(NSDate*)beginDate andDate:(NSDate*)end
     return YES;
 }
 
+- (BPKSimpleDate *)simpleDateFromDate:(NSDate *)date {
+    if(date == nil) {
+        return nil;
+    }
+    
+    NSDateComponents *components = [self.calendarView.gregorian components:NSCalendarUnitYear|NSCalendarUnitMonth|NSCalendarUnitDay
+                                                                  fromDate:date];
+    
+    return [[BPKSimpleDate alloc] initWithYear:components.year month:components.month day:components.day];
+}
+
+- (NSArray<BPKSimpleDate *> *)simpleDatesFromDates:(NSArray<NSDate *> *)dates {
+    NSMutableArray *simpleDates = [NSMutableArray new];
+    
+    for (NSDate *date in dates) {
+        [simpleDates addObject:[self simpleDateFromDate:date]];
+    }
+    
+    return [simpleDates copy];
+}
+
+-(NSDate *)dateFromSimpleDate:(BPKSimpleDate *)simpleDate {
+    if(simpleDate == nil) {
+        return nil;
+    }
+    
+    NSDateComponents *components = [[NSDateComponents alloc] init];
```

Comment:
`init` isn't documented are we sure that it correctly sets the timeZone property?